File: KarateKlub/KarateKlub.Api/Models/IzmireneClanarinePregledVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace KarateKlub.Api.Models
{
    public class IzmireneClanarinePregledVM
    {
        public class Row
        {
            public int clanarinaId;
            public int clanKlubaId;
            public string clanKluba;
            public string brojPriznanice;
            public string iznosBrojevima;
            public string mjestoUplate;
            public DateTime? datumUplate;
            public DateTime datumOd;
            public DateTime datumDo;
        }
        public List<Row> rows;
    }
}

File: Android aplikacija/KarateKlub/app/src/main/java/com/example/x/karateklub/area_blagajnik/model/IzmireneClanarinePregledVM.java
package com.example.x.karateklub.area_blagajnik.model;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

public class IzmireneClanarinePregledVM implements Serializable {
    public  static  class Row implements Serializable{
        public int clanarinaId;
        public int clanKlubaId;
        public String clanKluba;
        public String brojPriznanice;
        public String iznosBrojevima;
        public String mjestoUplate;
        public Date datumUplate;
        public Date datumOd;
        public Date datumDo;
    }
 public List<Row> rows;
}
